tools/deploy.py

```diff
@@ -178,8 +189,8 @@ def collect_manifest_data(job_data):
     manifest_data = {}
     for output_name, output_data in job_data["outputs"].items():
         # Filter out unrelated job outputs that don't start with "artifact-".
-        prefix, _, slug = output_name.partition("-")
-        if prefix != "artifact" or not slug:
+        prefix, _, artifact_slug = output_name.partition("-")
```

Comment:
Does this have an issue with the `-`s in the `git describe` output?

Reply:
That's unrelated, this parses the output names from here: https://github.com/Holzhaus/mixxx/blob/ci-deployment-cleanup/.github/workflows/build.yml#L122-L124